wildfly/unifiedpush-wildfly/Dockerfile

```diff
@@ -1,7 +1,11 @@
 # Use latest aerogear/wildfly image as the base
-FROM aerogear/wildfly
+FROM rhmap/wildfly
```

Comment:
We can switch this back when we build and push the `aerogear/wildfly` image and tag it as 1.1.x or similar (the one on master doesn't seem to start correctly with the 1.1.x branch version of UPS).